R/dataprep_lookup_in_pandora.R

```diff
@@ -40,3 +40,25 @@ namecol_value_from_id <- function(sidora_col_name, query_id, con, cache_dir = te
   return(res_vector)
   
 }
+
+#' convert_default_lookup
+#'
+#' A convenience function which simply transforms a given Pandora-Dataframe using all
+#' defined default lookups. Typically will convert a pandora back-end numeric ID to a 'human readable string' actually displayed on the pandora webpage.
+#' 
+#' @param df data.frame. A Sidora/Pandora-Dataframe with sidora column names.
+#' 
+#' @return The converted dataframe with lookup-columns replaced by the actual values.
+#'
+#'
+#' @export
+convert_default_lookup <- function(df) {
+  cols2update <- names(df[sidora.core::sidora_col_name_has_aux(names(df))])
+  return (df %>% 
```

Comment:
FYI R functions do not need explicit return statements. The function returns the output of the last expression.

Reply:
Ah right. OK, good to know.